CHANGELOG.md

```diff
@@ -12,25 +12,24 @@ vars:
   - The `int_zendesk__schedule_spine` model is now able to incorporate these schedule changes, making it possible for downstream models to reflect the most up-to-date schedule data.
     - Note this is only in effect when `using_schedule_histories` is true.
   - This improves granularity for Zendesk metrics related to agent availability, SLA tracking, and time-based performance analysis.
-### dbt_zendesk_source changes (see the [Release Notes](https://github.com/fivetran/dbt_zendesk_source/releases/tag/v0.13.0) for more details)
+#### dbt_zendesk_source changes (see the [Release Notes](https://github.com/fivetran/dbt_zendesk_source/releases/tag/v0.13.0) for more details)
 - Introduced the `stg_zendesk__audit_log` table for capturing schedule changes from Zendesk's audit log.
   - This model is disabled by default, to enable it set variable `using_schedule_histories` to `true` in `dbt_project.yml`.
 
 ## New Features
-- Holiday support: Users can now choose to disable holiday tracking by setting variable `using_holidays` to `false` in `dbt_project.yml`.
+- Holiday support: Users can now choose to disable holiday tracking, while continuing to use schedules, by setting variable `using_holidays` to `false` in `dbt_project.yml`.
 - New intermediate models have been introduced to streamline both the readability and maintainability:
-  - `int_zendesk__timezone_daylight`: A utility model that maintains a record of daylight savings adjustments for each time zone.
+  - [`int_zendesk__timezone_daylight`](https://github.com/fivetran/dbt_zendesk/blob/main/models/intermediate/int_zendesk__timezone_daylight.sql): A utility model that maintains a record of daylight savings adjustments for each time zone.
```

Comment:
```suggestion
  - [`int_zendesk__timezone_daylight`](https://github.com/fivetran/dbt_zendesk/blob/main/models/utils/int_zendesk__timezone_daylight.sql): A utility model that maintains a record of daylight savings adjustments for each time zone.
```
unless you wanna move [it](https://github.com/fivetran/dbt_zendesk/blob/feature/historical-schedules/models/utils/int_zendesk__timezone_daylight.sql) to intermediate?

Reply:
Ooo good catch. Updated!